src/analyses/flagModes.ml

```diff
@@ -52,7 +52,7 @@ struct
 
   let branch ctx (exp:exp) (tv:bool) : D.t =
     if ctx.local = D.bot() then ctx.local else begin
-      match%distr (tv, (constFold false exp)) with
+      match (tv, (constFold false exp)) with (* TODO make match%distr (ppx_distr_guards) work, otherwise the below warnings 57 are actually bugs *)
```

Comment:
TODO: migrate [ppx_distr_guards](https://github.com/vogler/ppx_distr_guards/blob/master/ppx_distr_guards.ml) to ocaml-migrate-parsetree.driver as mentioned in [dune docs](https://dune.readthedocs.io/en/stable/concepts.html#preprocessing-with-ppx-rewriters): [example](https://github.com/ocaml-ppx/ocaml-migrate-parsetree/blob/master/examples/omp_ppx_here/ppx_here.ml).

Reply:
waiting for https://github.com/ocaml/opam-repository/pull/15897